src/main/java/com/minecolonies/entity/ai/util/ChatSpamFilter.java

```diff
@@ -17,37 +17,36 @@
      * 20 Ticks per second.
      * 30 Seconds.
      */
-    protected static final int BASE_TIMEOUT  = 30 * 20;
+    static final int BASE_TIMEOUT = 30 * 20;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Explicitly declare the visibility for "BASE_TIMEOUT". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2039)